shared/actions/favorite.js

```diff
@@ -79,6 +79,7 @@ const folderToProps = (folders: Array<Folder>, username: string = ''): FolderPro
     showingPrivate: true,
     smallMode: false,
     showComingSoon: !flags.tabFoldersEnabled,
+    username,
```

Comment:
and actually this PR has it pulling it from config already, so we already don't need this right?